terraform-setup/global/iam/main.tf

```diff
@@ -13,12 +13,12 @@ provider "aws" {
         principals {
         type        = "AWS"
       identifiers = [
```

Comment:
Why do you need this ?

Reply:
This is to allow the IAM users assume their respective IAM roles